### DISCUSSION TITLE
当你使用授权服务时，无法访问 github.io 的处理办法

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1770] [重要通知]
## 问题描述

RT，发生问题的授权包括：

- Dropbox
- Pocket
- Onenote
- Google Drive
- 语雀

## 问题原因

`ping kenshin.github.io` 如果不是解析到 **185.199.108.153** 的话，则说明解析错误。

## 解决方案

使用 `nslookup kenshin.github.io` 后再次测试 `ping kenshin.github.io` 如 IP 地址如上所示，则解决问题。

## 关联问题

https://github.com/Kenshin/simpread/issues/986

**Comment by binsee** [comment]
DNS污染，可参考以下命令解析正确ip：

```cmd
# nslookup kenshin.github.io 119.29.29.29
Server:  pdns.dnspod.cn
Address:  119.29.29.29

Non-authoritative answer:
Name:    kenshin.github.io
Addresses:  185.199.109.153
          185.199.111.153
          185.199.108.153
          185.199.110.153
```

然后再对比 `nslookup kenshin.github.io` 命令通过默认dns解析的结果
